Template10 (Library)/Behaviors/NavToPageAction.cs

```diff
@@ -26,15 +26,7 @@ public object Execute(object sender, object parameter)
             if (nav == null)
                 throw new NullReferenceException($"Cannot find NavigationService for {Frame.ToString()}.");
 
-            var metadataProvider = Application.Current as IXamlMetadataProvider;
-            if (metadataProvider == null)
-                return false;
-
-            var pagetype = metadataProvider.GetXamlType(Page as Type);
-            if (pagetype == null)
-                throw new NullReferenceException($"Cannot find TargetPage:{Page}");
-
-            nav.Navigate(pagetype as Type, Parameter, InfoOverride);
+            nav.Navigate(Page as Type, Parameter, InfoOverride);
```

Comment:
If the value of Page is a string because it is a view in a remote assembly, how does "as Type" work? Could you explain why you remove this logic?